packages/sign_in_with_apple/test/sign_in_with_apple_button/sign_in_with_apple_button_test.dart

```diff
@@ -19,8 +19,10 @@ Future<void> main() async {
   final fontLoader = FontLoader('.SF Pro Text')..addFont(fontData);
   await fontLoader.load();
 
+  // The tests should only be run on macOS system
+  // On other systems small differences can lead to failing goldens (e.g. text rendering)
   if (!Platform.isMacOS) {
-    return;
+    throw UnsupportedError('Widget tests should only be run on macOS machines');
```

Comment:
I would think this should lead to an error instead of silently skipping the tests

Reply:
But how about CI then? I think the simplest would be just `skip: !Platform.isMacOS` (I think Flutter has similar patterns for some).

Then at least it shows up as being skipped :)